src/libos/src/fs/fs_ops/statfs.rs

```diff
@@ -18,12 +29,23 @@ pub fn do_statfs(path: &str) -> Result<Statfs> {
         let fs = current.fs().read().unwrap();
         fs.lookup_inode(path)?
     };
-    let statfs = Statfs::from(inode.fs().info());
+    let statfs = {
```

Comment:
Is it better to combine the do_statfs and do_fstatfs together: one invoke another, or both of them use a common function?

Reply:
Done